WORKSPACE

```diff
@@ -21,19 +29,21 @@ load(
 
 swift_rules_dependencies()
 
+load(
+    "@build_bazel_rules_swift//swift:extras.bzl",
+    "swift_rules_extra_dependencies",
+)
+
+swift_rules_extra_dependencies()
+
 load(
     "@build_bazel_apple_support//lib:repositories.bzl",
     "apple_support_dependencies",
 )
 
 apple_support_dependencies()
 
-load(
-    "@com_google_protobuf//:protobuf_deps.bzl",
-    "protobuf_deps",
-)
```

Comment:
Mind clarifying why this is being removed?

Reply:
This is for rules swift and now loaded through the new "extras" function